automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/NotebooksListPage.scala

```diff
@@ -62,11 +61,14 @@ class NotebooksListPage(override val url: String)(override implicit val authToke
 
   def withNewNotebook[T](kernel: Kernel = PySpark2)(testCode: NotebookPage => T): T = {
     switchToNewTab {
-      click on (await enabled(newButton, 30))
-      click on (await enabled cssSelector(kernel.cssSelectorString))
```

Comment:
`await enabled` is flaky as `div` web element is not `type=button` and doesn't have `enabled` state.